ibis/backends/tests/test_dot_sql.py

```diff
@@ -237,7 +237,7 @@ def test_dot_sql_reuse_alias_with_different_types(backend, alltypes, df):
     backend.assert_series_equal(foo2.x.execute(), expected2)
 
 
-_NO_SQLGLOT_DIALECT = {"pandas", "dask", "datafusion", "druid"}
+_NO_SQLGLOT_DIALECT = {"pandas", "dask", "datafusion", "druid", "flink"}
```

Comment:
Was this change needed here?

Reply:
Nope, that was from an unrelated thing that I need to remove.

Reply:
Hm, actually this does seem necessary. Otherwise dot sql tests attempt to check for the flink sql dialect and fail.